lib/wovnrb/services/html_converter.rb

```diff
@@ -206,7 +207,9 @@ def data_wovnio
         "langCodeAliases=#{lang_code_aliases_json}",
         "langParamName=#{lang_param_name}",
         "version=WOVN.rb_#{VERSION}"
-      ].join('&')
+      ]
+      result << "customDomainLangs=#{custom_domain_langs}" unless custom_domain_langs == '{}'
```

Comment:
can you check this earlier instead of comparing against a string?
e.g.
```
custom_domain_langs = @store.custom_domain_langs.to_html_swapper_hash
result << JSON.generate(custom_domain_langs) unless custom_domain_langs.empty?
```

Reply:
I don't think it matters if we have just `customDomainLangs=` in `data-wovnio` (ref. [insert_hreflang_expected.html (Wovn.cs)](https://github.com/WOVNio/Wovn.cs/blob/cae38c052811d5050347931775de17da9929d7b7/wovn.cs.Tests/Fixtures/BasicHtml/insert_hreflang_expected.html#L3)). Maybe we can just remove the condition. WDYT?

Reply:
no it doesn't matter in practice. But I think it's probably cleaner to skip the empty params which we aren't consistent with.

Reply:
Understood. Fixed in ae308849873b5250ca9082750879ce9ee38ff4c3.